db/ens.go

```diff
@@ -541,8 +545,8 @@ func removeEnsName(client *ethclient.Client, name string) error {
 		ens_name = $1
 	;`, name)
 	if err != nil {
-		utils.LogError(err, fmt.Errorf("error deleting ens name [%v]", name), 0)
-		return err
+		logger.Warnf("error deleting ens name [%v]: %v", name, err)
+		return nil
```

Comment:
I suggest to keep the error and only log a warning for the error in question (i.e. `ERROR: invalid byte sequence for encoding "UTF8": 0x00 (SQLSTATE 22021)` . This would be a less intrusive change.